.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 matrix:
   include:
-    - node_js: "8"
```

Comment:
let's leave node 8 in here until we officially drop support for it in our next major?

I'm with ya on dropping it, just don't want to publish a new major at this exact moment :)